tools/fuchsia/build_fuchsia_artifacts.py

```diff
@@ -91,7 +94,7 @@ def ProcessCIPDPakcage(upload, engine_version):
   cipd_yaml = os.path.join(_script_dir, 'fuchsia.cipd.yaml')
   CopyFiles(cipd_yaml, os.path.join(_bucket_directory, 'fuchsia.cipd.yaml'))
 
-  if upload:
+  if upload and IsLinux():
```

Comment:
Are we still building the mac and linux Dart packages?

Reply:
We will still build all the specified target. We even run `cipd pkg-build` command will still be executed. It's the else clause. We only restrict uploading the packages to linux.